packages/directory/addon/templates/components/dir-search-bar.hbs

```diff
@@ -5,4 +5,8 @@
     class='dir-search-bar__input'
     key-up=(action this.attrs.searchFor)
 }}
-{{navi-icon 'search' class='dir-search-bar__search-icon'}}
\ No newline at end of file
+{{navi-icon 'search' class='dir-search-bar__search-icon'}}
+{{navi-icon 'times' 
+    class=(concat 'dir-search-bar__clear-icon' (if (not-eq query.length 0) ' dir-search-bar__clear-icon--show'))
+    click=(action (mut query) '')
```

Comment:
Should this `clear search` functionality be tested?

Reply:
done